public/index.php

```diff
@@ -1,9 +1,11 @@
 <?php
 
+declare(strict_types=1);
+
 use App\Kernel;
 
 require_once dirname(__DIR__).'/vendor/autoload_runtime.php';
 
-return function (array $context) {
+return static function (array $context) {
```

Comment:
This is a recommendation of the EA plugin : `// [EA] This closure can be declared as static (better scoping; in some cases can improve performance).` I suppose this is a micro optimization.

Reply:
Can we stay in sync with upstream Symfony? Maybe can you propose this change to Symfony directly?